scripts/deploy.ts

```diff
@@ -0,0 +1,9 @@
+async function main() {
```

Comment:
I would not add this script, as we should do deployments via `hardhat-deploy` (for an example see https://github.com/safe-global/safe-contracts/blob/main/src/deploy/deploy_handlers.ts)